resources/views/layouts/partials/header/autocomplete.blade.php

```diff
@@ -9,7 +9,7 @@ class="contents"
             v-cloak
         >
             <div class="contents">
-                <ais-configure :hitsPerPage="3" />
+                <ais-configure :hits-per-page.camel="3" />
```

Comment:
How come this was changed? Did the other version not work?

Reply:
Correct, `hitsPerPage` with camelCase is only possible in `.vue` or `.js`, html does not allow camelCase in it's arguments.
`:hits-per-page` did not seem to work either. `hits-per-page.camel` restored functionality back.